source/_patterns/01-atoms/dropdown/demo/dropdowns.md

```diff
@@ -0,0 +1,16 @@
+## Variables
```

Comment:
Looking forward to this change. I'm assuming we'll be moving toward this in general?